docs/wtf-is-metagame/300-of-metagame.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: 💪 The 300 of MetaGame
+title: 💪 The 300 of MetaGame.md
```

Comment:
So sorry @metaverde we don't want the extension in the title. 😄 Only on the file itself as you have done.